.github/workflows/gh-actions-release.yml

```diff
@@ -83,5 +95,6 @@ jobs:
       uses: skx/github-action-publish-binaries@master
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        INSTALLER_EXT: ${{ matrix.INSTALLER_EXT }}
       with:
-        args: 'packages/Vega-Strike_v*.*'
+        args: 'packages/*.${INSTALLER_EXT}'
```

Comment:
We should have another one that manages the tarballs; and just one of them - we don't need that for every platform.
GH provides the ZIP and TGZ files automatically; so we just need to add the TBZ2 and TXZ files; would probably be good to do that as a separate job line so that it could be as platform agnostic as possible too...so may be that should be a separate PR.

Reply:
Pardon me, but I don't understand why we need so many different tarball formats. What's wrong with ZIP and TGZ?

Reply:
Zip is primarily used by Windows folks.
Traditionally open source used tgz and then tbz2 and more recently txz; however, it's typical that folks like us would provide all three so each distro can select the format they use.

Some Linux folks will only grab tgz; others only tbz2; and others yet only txz. so if we want to maintain a diverse set of users we'll want to maintain a diverse selection of archives formats too.

As we grow and get more integrated with distros we'll be able to refine that to what the distro level maintainers will use. We don't quite get the luxury yet that way while still achieving our goals in expanding the user and dev base.

Reply:
Hmm. I see.

And what should be in these tarballs? Binaries? Source? Both?

Reply:
Oh, I think I see. We need a single "source tarballs" build in the matrix. That would build TBZ2 and TXZ. Then we **also** need all 4 tarball formats with the binaries for unknown/not yet supported Linux distributions.

Moreover, for the Linux distros we support now, we should build **only** .RPM or **only** .DEB packages, as applicable.

Does that sound right? Take a look at what I've done in #404 , targeting the master branch, and let me know if it makes sense to you, @BenjamenMeyer .

Reply:
Yep. Sounds about right.

Reply:
OK, I think I'm done implementing these changes. Have a look when you get a chance.